### ISSUE TITLE
How do I find the correct typeHandlerVersion!

### ISSUE THREAD

**Issue by scott1138** [Microsoft/OMS-Agent-for-Linux#793]
The readme.md says the latest version is 1.4, the MS Docs site says in the example to use 1.7, but the chart says 1.7.9! https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/oms-linux

The output of Get-AzureRmVMExtensionImage -Location centralus -PublisherName Microsoft.EnterpriseCloud.Monitoring -Type Om
sAgentForLinux also shows 1.7.9 for the version.

I tried searching this repo but didn't find any reference to typeHandlerVersion.  : (

1.7 worked but I can't find an explanation for this anywhere and it would be nice to know how I get the correct typeHandlerVersion when we update templates.  Can anyone enlighten me?  Apologies in advance if I missed something obvious.

**Comment by scott1138** [comment]
Anyone? :(

**Comment by shpimpal** [comment]
Hi, please use 1.7.
1.7.9 is the latest extension version and the command needs first two numbers of it, the third number is always picked up latest.

We probably missed updating readme, but I could not find the place where it says 1.4. Could you please point to that ? Thanks.

**Comment by scott1138** [comment]
Sorry, I should have copied the link for that into my original comment.  Now i can't find it.

The 1.7 makes sense, but maybe add to the documentation that only the major.minor version is excepted and any additional version information is assumed to be the latest in the docs.  

As a future enhancement it would be nice to leave that off entirely and just have the latest version installed.  If I read the documentation correctly it auto-updates itself anyway.

**Comment by akramwaseem** [comment]
@shpimpal  -- Does the handler version change for new release which is 1.8 ?
Probably need to update that too.

**Comment by shpimpal** [comment]
yes, it will change too. The extension is being deployed to all the regions. Once done, we'll update the docs.

**Comment by karlomedallo** [comment]
@shpimpal the typeHandlerVersion was mentioned in one of the samples here: https://github.com/Azure/azure-quickstart-templates/blob/master/201-oms-extension-ubuntu-vm/azuredeploy.json

This is from: https://docs.microsoft.com/en-us/azure/azure-monitor/platform/template-workspace-configuration#example-resource-manager-templates

**Comment by dawsonrs** [comment]
@shpimpal  FYI, a typeHandlerVersion of 1.4 is also mentioned at:
https://github.com/Azure/azure-linux-extensions/tree/master/OmsAgent
(section 2.3)

**Comment by johncrim** [comment]
Is a new typeHandlerVersion required for 1.9.1 ? The relationship between typeHandlerVersion and versions listed here:
https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/oms-linux
should be explained on this page.

**Comment by hestolz** [comment]
Thank you for your submission. As [previously announced](https://azure.microsoft.com/en-us/updates/were-retiring-the-log-analytics-agent-in-azure-monitor-on-31-august-2024/), the Log Analytics agent has been **deprecatated** and has no support as of **August 31, 2024.** If you use the Log Analytics agent to ingest data to Azure Monitor, [migrate now to the new Azure Monitor agent](https://docs.microsoft.com/en-us/azure/azure-monitor/agents/azure-monitor-agent-migration). As part of repo archival, open issues and pull requests will be closed.
